misc/object-conditions/basics.rst

```diff
@@ -1,443 +1,438 @@
-          agent or customer updates the ticket, this will be the organization.
-          If this was not a human interaction, Zammad will use no organization.
-          This may have unexpected results!*
-      * | specific organization
-        | *Select one or more organizations*
-      * | not set (not defined)
-
-   .. container:: cfloat-clear
-
-      X
-
-Owner *(Scope: Ticket)*
-   .. container:: cfloat-left
-
-      | What ticket owner is affected (or not)?
-      | Available matching: *is*, *is not* or *has changed*
-
-   .. container:: cfloat-right
-
-      Offered values (multiple choice):
-
-      * | current user
-        | *This is the user that caused the trigger run. If your agent or
-          customer updates the ticket, this will be the user. If this was not
-          a human interaction, Zammad will use the system user. This may have
-          unexpected results!*
-      * | specific user
-        | *Select one or more owners*
-      * | not set (not defined)
-
-   .. container:: cfloat-clear
-
-      X
-
-Sender *(Scope: Ticket article)*
-   .. container:: cfloat-left
-
-      | What user role does the sender of the article have (or not)?
-      | Available matching: *is* or *is not*
-
-   .. container:: cfloat-right
-
-      Determine the sender of the message: System, Agent or Customer.
-
-   .. container:: cfloat-clear
-
-      X
-
-State *(Scope: Ticket)*
-   .. warning::
-
-         Zammad behaves inconsistent in between certain ticket condition
-         dialogs - if you can see ticket states only in parts of the
-         admin menu and not in the front-end, your state *is not visible*.
-
-         :docs:`The system documentation </admin/console/working-on-tickets.html#make-new-states-available-to-ui>`
-         has you covered.
-
-   .. container:: cfloat-left
-
-      | What ticket state is affected (or not)?
-      | Available matching: *is*, *is not* or *has changed*
-
-   .. container:: cfloat-right
-
-      Offers all configured and *visible* ticket states in Zammad.
-
-   .. container:: cfloat-clear
-
-      X
-
-Subscribe *(Scope: Ticket)*
-   This affects ticket subscriptions / mentions by and for agents.
-
-   .. container:: cfloat-left
-
-      | What ticket subscribers
-        (:user-docs:`notifications </advanced/suggested-workflows.html#mentions-the-subscribe-button>`)
-        are affected (or not)?
-      | Available matching: *is* or *is not*
-
-   .. container:: cfloat-right
-
-      Offered values (multiple choice):
-
-      * | current user
-        | *This is the user that caused the trigger run.
-          Only can affect agents.*
-      * | specific user
-        | *Affects one or more specific users that have subscribed to the
-          ticket.*
-      * | not set (not defined)
-
-   .. container:: cfloat-clear
-
-      X
-
-Tags *(Scope: Ticket)*
-   Additional tags can be present in the ticket without issues.
+Object Conditions
```

Comment:
For the parent page, "Getting started" should be title cased

Reply:
Also the "Object conditions" itself should be title cased.

Reply:
Thanks for the hint! I reworked the page as well and removed the `Getting started` section.